### PR TITLE
Airport challenge submission - 1st attempt

### PR DESCRIPTION
**Pull request by MPFekete**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,199 @@
 Airport Challenge
 =================
```
Comment (MPFekete) [inline README.md:3]: I had to go back and reformat my domain model tables to make them readable, so definitely need to get more familiar with using markdown!

I also completely missed out creating a Runner as part of the testing process and this seriously undermined the efficacy of my tests - I had passing tests but unexpected Runner results when I did eventualy realise my oversight, and spent a lot of time trying to figure out what was happening. Comment in airport.js 17 is an example of this. I would make sure to create my Runner alongside my tests if doing this again.

I can also see in review that I have not really covered enough edge cases, and need to make sure edge cases are given more attention in my TDD going forward. If I had written more edge case tests I would probably have identified the unexpected Runner outcomes a lot earlier as well, saving myself a lot of time.

File: src/Plane.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Plane {
```
Comment (MPFekete) [inline src/Plane.js:1]: Added name constructor to Plane so that names could be set on the instances (I had initally been trying to use names in the Runner without setting a name constructor...)
